service/build.gradle

```diff
@@ -24,6 +25,7 @@ dependencies {
     implementation 'bio.terra:datarepo-client:1.313.0-SNAPSHOT'
     implementation 'javax.ws.rs:javax.ws.rs-api:2.1.1'
     implementation 'org.postgresql:postgresql:42.3.3'
+    implementation 'com.google.code.gson:gson:2.9.1'
```

Comment:
It's up to you, but I'd avoid using `gson` in a spring app since spring already brings in jackson as a dependency, and jackson does (pretty much) everything gson does.

Reply:
Good catch... I think this was leftover from when the WDS client wasn't importing for me and I tried writing my own (I quickly backed away from that idea)!